src/sidebar/test/media-embedder-test.js

```diff
@@ -90,6 +90,28 @@ describe('media-embedder', function () {
     });
   });
 
+  it('replaces internet archive links with iframes', function () {
+    var urls = [
+      'https://archive.org/details/PATH/start/360/end/420.3',
```

Comment:
Do you know where I can find examples of URLs with the start and end parameters embedded in the path like this? Using Google I found this page (https://archive.org/details/PopcornArchive-Isis) where there is a URL (https://archive.org/details/KNTV_20160927_010000_NBC_News_Special_2016_Presidential_Debate_1#start/1560/end/1620) with the start/end times in the _fragment_ but not in the path.

Reply:
I tested this with https://archive.org/details/sims_evolved_virtual_creatures_1994/start/360/end/420.3 and also tried https://archive.org/details/sims_evolved_virtual_creatures_1994#start/360/end/420.3 . Neither worked.

Using `start=<seconds>` and `end=<seconds>` query params in the URL did work.

Unless you can point me at a working example of setting the start and end time of the clip this way, I suggest we remove this test case and example for now and require users to use start and end query params.

Reply:
https://archive.org/details/KNTV_20160927_010000_NBC_News_Special_2016_Presidential_Debate_1#start/1560/end/1620  redirects to https://archive.org/details/KNTV_20160927_010000_NBC_News_Special_2016_Presidential_Debate_1/start/1560/end/1620

However https://archive.org/details/sims_evolved_virtual_creatures_1994/start/360/end/420.3 is an interesting find! It looks like the TV News Archive may be a subset of IA-hosted videos, with behavior that does not apply more broadly. 

I'll ask the IA folks to comment here on whether/how we might want to restrict this to that subset, and if so, how to identify it.